src/components/Login/Login.component.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useRef } from 'react';
+import { Modal } from 'bootstrap';
+import Styled from './Login.styled';
+import loginAPI from '../../utils/mocks/login.api';
+import { useAuth } from '../../providers/Auth.provider';
+
+function Login() {
+  const { state, dispatch } = useAuth();
+  const { authenticated } = state;
+  const userNameField = useRef();
+  const passwordField = useRef();
+
+  const sendLogin = async (event) => {
+    event.preventDefault();
+    const userName = userNameField.current.value;
```

Comment:
For controlled inputs like username and password in this case, normally in React using local state would be the best practice. Something like:

```js
const [usernameValue, setUsernameValue] = useState('')
```

Where you `setUsernameValue` in the `onChange` event for the input.

So you can have something like this:

```js
...
const [usernameValue, setUsernameValue] = useState('')

return (
<Styled.Input
    required
    type="text"
    placeholder="User Name"
    id="username"
    value={usernameValue}
    onChange={(e) => {setUsernameValue(e.target.value}}
  />
);
```